src/main/java/io/adobe/cloudmanager/Region.java

```diff
@@ -0,0 +1,59 @@
+package io.adobe.cloudmanager;
+
+/*-
+ * #%L
+ * Adobe Cloud Manager Client Library
+ * %%
+ * Copyright (C) 2020 - 2023 Adobe Inc.
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import lombok.Getter;
+
+/**
+ * Represents possible regions for deployments, network, and other configurations.
+ */
+@Getter
+public enum Region {
+  SOUTHEAST_AUSTRALIA("aus5"),
```

Comment:
Though this doesn't seem likely to change any time soon, why do we need specific cluster names with numbers? It might make more sense to support pattern matching here or at least string.startsWith() and just look at the string prefixes in case one day we switch from, say, va7 to va6.  It's unlikely but it's also probably not necessary for the code to be super-specific since most of these prefixes are country codes with the exception of US states.

Reply:
This is what is in the API spec, so i need to pass the literal value when. And this is the only place they exist (they aren't in the CM API's yaml spec)